src/database/organizationSchema.ts

```diff
@@ -15,7 +15,7 @@ const OrganizationSchema = new Schema({
   reimbursements: {
     type: [{ type: Schema.Types.ObjectId, ref: "Reimbursement" }],
     required: [
-      true,
+      false,
```

Comment:
is it possible to make this still required but initialize it to an empty array upon creation? otherwise when we try to do array operations in the future it may be null